recipes/opentdf-client/all/conanfile.py

```diff
@@ -60,6 +60,8 @@ def validate(self):
             if Version(self.settings.compiler.version) < min_version:
                 raise ConanInvalidConfiguration(f"{self.name} requires C++{self._minimum_cpp_standard} support."
                                                  "The current compiler {self.settings.compiler} {self.settings.compiler.version} does not support it.")
+        if is_msvc_static_runtime(self):
```

Comment:
```suggestion
        if is_msvc_static_runtime(self) and Version(self.version) >= "1.1.5":
```


The previous version was working with these runtimes, right?

Reply:
No, not really.  The compiler version check (17) prevented anything < 1.1.5 from being built on windows on conan-center.

Reply:
Oh, you are right. Side note.- The pipeline for v2 will include VisualStudio 17, but the one for v1 doesn't (it's been decided that way to encourage migration to v2).